app/src/App.js

```diff
@@ -0,0 +1,18 @@
+import { Fragment } from 'react'
+
+import Nav from './components/Nav'
+import Box from './components/Box';
+
+function App() {
+  return (
+    <Fragment>
```

Comment:
The shorthand for Fragements from react is just `<></>` so in this case it would be like

```jsx
<>
 <Nav/>
 <Box/>
</>
```